projects/OdroidXU3/bootloader/boot.ini

```diff
@@ -12,23 +12,9 @@ setenv macaddr "00:1e:06:61:7a:39"
 # Boot Specific Stuff
 #
 #------------------------------------------------------------------------------------------------------
-setenv bootrootfs "boot=/dev/mmcblk0p1 disk=/dev/mmcblk0p2 ro"
+setenv bootrootfs "console=tty1 console=ttySAC2,115200n8 boot=LABEL=LAKKA disk=LABEL=LAKKA_DISK rootwait ro"
 
 #------------------------------------------------------------------------------------------------------
-#
-# Console Setup
-#
-#------------------------------------------------------------------------------------------------------
-setenv console "console=ttySAC2,115200n8"
-
-#------------------------------------------------------------------------------------------------------
-#
-# OpenELEC variables
-#
-# Setup the OpenELEC options
-#   valid values are: progress ssh debugging quiet break verbose
-#------------------------------------------------------------------------------------------------------
-setenv openelec "quiet consoleblank=0 loglevel=2 vt.global_cursor_default=0"
```

Comment:
Note: `consoleblank=0 loglevel=2 vt.global_cursor_default=0` is needed

Reply:
can you please test this commit:
https://github.com/Ntemis/Lakka-LibreELEC/commit/ba5047292d01dc1e5a7df5734db29692aa3f6b2f

Reply:
Sure, but I'm getting a

    /home/kivutar/Lakka-LibreELEC/build.Lakka-OdroidXU3.arm-2.1-devel/toolchain/lib/gcc/armv7ve-libreelec-linux-gnueabi/6.2.0/../../../../armv7ve-libreelec-linux-gnueabi/bin/ld.gold: error: cannot find -lGLESv1_CM
    collect2: error: ld returned 1 exit status

When linking RA